test/plugins/features/movies/controller.test.js

```diff
@@ -23,5 +24,55 @@ describe('movie controller', () => {
 
   });
 
+  describe('list', () => {
+
+    beforeEach(() => {
+      return Knex.raw('TRUNCATE movies CASCADE;');
+    });
+
+    beforeEach(() => {
```

Comment:
you can combine these two beforeEach blocks and use .then to chain them

`
beforeEach(() => {
     return Knex.raw('TRUNCATE movies CASCADE;')
     .then(() => {
        return Knex('movies').insert(movie_list);
     });
 });
`